presto-cli/src/main/java/com/facebook/presto/cli/StatusPrinter.java

```diff
@@ -164,10 +164,9 @@ private String autoFormatMetricValue(RuntimeUnit unit, long value)
 
     public void printFinalInfo()
     {
-        Duration wallTime = nanosSince(start);
-
         QueryStatusInfo results = client.finalStatusInfo();
         StatementStats stats = results.getStats();
+        Duration wallTime = Duration.succinctDuration(stats.getElapsedTimeMillis(), MILLISECONDS);
```

Comment:
@aaneja What if the time is above one second? One minute or an hour? How would the time be displayed? Could you show an example for each of such cases?

Reply:
Actually it may not be necessary to run a query > 1 hour, I just want to know if the display for such cases would be in milliseconds or not.

Reply:
We use `FormatUtils.formatTime` to get the string to print. Examples for different durations are in [this test case](https://github.com/prestodb/presto/blob/e14ba2b4e6bded261a3c71656098c8df323309bd/presto-cli/src/test/java/com/facebook/presto/cli/TestFormatUtils.java#L26-L33)